apps/site/src/pages/api/external-service-method.api.ts

```diff
@@ -45,8 +45,22 @@ export default createApiKeyRequiredHandler<
     } catch (error) {
       const axiosError = error as AxiosError;
```

Comment:
Do we want to introduce and use this here

```ts
const isErrorAxiosError = (error: Error): error is AxiosError =>
  (error as AxiosError).isAxiosError;
```

Reply:
Yes